### PR TITLE
Add coverage RDD

### PR DESCRIPTION
**Pull request by Georgehe4**
Pending resolving loadParquet issues

### CODE REVIEW

File: mango-cli/src/main/webapp/resources/pileup.js [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit faaf7b789966e5d289cd0e12bdec9bef10266a99
+Subproject commit 54747798e4108ff6d1b6b40905356347d7a5f077
```
Comment (akmorrow13) [inline mango-cli/src/main/webapp/resources/pileup.js]: What changed in pileup?

File: mango-core/src/main/scala/org/bdgenomics/mango/models/CoverageMaterialization.scala [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+   * Loads ADAM data using predicate pushdowns
+   *
+   * @param sc SparkContext
+   * @param region Region to load
+   * @param fp filepath to load from
+   * @return RDD of data from the file over specified ReferenceRegion
+   */
+  def loadAdam(sc: SparkContext, region: ReferenceRegion, fp: String): CoverageRDD = {
+    val predicate = ((LongColumn("end") <= region.end) && (LongColumn("start") >= region.start) && (BinaryColumn("contigName") === region.referenceName))
+    //sc.loadCoverage(fp)
```
Comment (akmorrow13) [inline mango-core/src/main/scala/org/bdgenomics/mango/models/CoverageMaterialization.scala]: remove comment if not needed

File: mango-core/src/test/scala/org/bdgenomics/mango/models/CoverageMaterializationSuite.scala [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+  implicit val formats = DefaultFormats
+
+  val coverageFileName = "mouse.bed"
+  val coverageFile = resourcePath(coverageFileName)
+  val key = LazyMaterialization.filterKeyFromFile(coverageFileName)
+
+  val dict = new SequenceDictionary(Vector(SequenceRecord("chrM", 16699L)))
+
+  sparkTest("test") {
```
Comment (akmorrow13) [inline mango-core/src/test/scala/org/bdgenomics/mango/models/CoverageMaterializationSuite.scala]: add description for test

File: mango-core/src/test/scala/org/bdgenomics/mango/models/CoverageMaterializationSuite.scala [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    val rdd = sc.loadCoverage(coverageFile)
+    println(rdd.rdd.count)
+    // TODO: Complete coverage tests once issues with loadParquetAlignments are resolved
+  }
```
Comment (akmorrow13) [inline mango-core/src/test/scala/org/bdgenomics/mango/models/CoverageMaterializationSuite.scala]: TODO, assertion

File: mango-core/src/test/scala/org/bdgenomics/mango/models/CoverageMaterializationSuite.scala [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+import net.liftweb.json._
+import org.bdgenomics.adam.models.{ ReferenceRegion, SequenceDictionary, SequenceRecord }
+import org.bdgenomics.mango.util.MangoFunSuite
+import org.bdgenomics.adam.rdd.ADAMContext._
+
+class CoverageMaterializationSuite extends MangoFunSuite {
+
+  implicit val formats = DefaultFormats
+
+  val coverageFileName = "mouse.bed"
```
Comment (akmorrow13) [inline mango-core/src/test/scala/org/bdgenomics/mango/models/CoverageMaterializationSuite.scala]: more descriptive filename  (add coverage if this is what it is storing)

**Comment by akmorrow13** [comment]
Looks like there are a lot of unnecessary resources here.
mango-cli/src/test/resources/mouse_chrM.coverage.bam/
mango-core/src/test/resources/mouse_chrM.coverage.adam
 mango-core/src/test/resources/mouse_chrM.coverage.a.adam

**Comment by Georgehe4** [comment]
Moved to #232

**Pull request closed without merging**